src/__tests__/handleSubmit.spec.js

```diff
@@ -16,18 +16,19 @@ describe('handleSubmit', () => {
     const props = { startSubmit, stopSubmit, touch, setSubmitFailed, setSubmitSucceeded, values }
 
     handleSubmit(submit, props, false, asyncValidate, [ 'foo', 'baz' ])
-
-    expect(submit).toNotHaveBeenCalled()
-    expect(startSubmit).toNotHaveBeenCalled()
-    expect(stopSubmit).toNotHaveBeenCalled()
-    expect(touch)
-      .toHaveBeenCalled()
-      .toHaveBeenCalledWith('foo', 'baz')
-    expect(asyncValidate).toNotHaveBeenCalled()
-    expect(setSubmitSucceeded).toNotHaveBeenCalled()
-    expect(setSubmitFailed)
-      .toHaveBeenCalled()
-      .toHaveBeenCalledWith('foo', 'baz')
+      .then(() => {
```

Comment:
most of the changes in the tests are just whitespace, because I had to move the expect's into then()